README.md

```diff
@@ -53,7 +57,18 @@ encourage project teams to share progress updates or post questions in these
 channels:
 
 - [Email list](https://lists.cncf.io/g/cncf-sig-contributor-strategy)
+  - The mailing list is used for decisions, announcements, and broad communications
+  on record for the entire SIG.
+  - Calendar invites will be sent to this address. You can get the calendar
```

Comment:
It's not the same meeting I assume (contributor growth vs sig meeting itself).

Reply:
oh sorry yeah, this update was for the general group. i saw it and made the edit while i was in there. :) i lumped it into one PR with the addition of the wg which should have been two.

Reply:
contrib growth doesn't have meetings (right now anyway) outside of our general